eng/tools/lint-diff/src/runChecks.ts

```diff
@@ -23,6 +24,13 @@ export async function runChecks(
   for (const [readme, tags] of runList.entries()) {
     const changedFilePath = join(path, readme);
 
+    // TEST: Ignore readme.md if no "input-file"s
+    // Needed for extra resource-manager/readme.md required for RPaaS
+    const readmeContent = await readFile(changedFilePath, { encoding: "utf-8" });
+    if (!readmeContent.includes("input-file")) {
+      continue;
+    }
+
```

Comment:
Skip should be happening in `processChanges.ts` where changed readme.md files are added to the list with no tags to scan: https://github.com/Azure/azure-rest-api-specs/blob/main/eng/tools/lint-diff/src/processChanges.ts#L135-L148

In this case, if the file content doesn't contain `input-file` then `continue`.